src/debuginfo.cpp

```diff
@@ -728,50 +719,44 @@ static void jl_getDylibFunctionInfo(char **name, char **filename, size_t *line,
         if (SymFromAddr(GetCurrentProcess(), dwAddress, &dwDisplacement64,
                         pSymbol)) {
             // SymFromAddr returned success
-            jl_copy_str(name, pSymbol->Name);
-            saddr = (void*)(uintptr_t)pSymbol->Address;
+            if (name)
+                jl_copy_str(name, pSymbol->Name);
+            if (saddr)
+                *saddr = (void*)(uintptr_t)pSymbol->Address;
+            if (symsize)
+                *symsize = pSymbol->Size;
```

Comment:
this value probably will usually be wrong (if it is even available, this branch can fails pretty often too)